packages/explorer/src/app/(explorer)/[chainName]/page.tsx

```diff
@@ -6,6 +6,6 @@ type Props = {
   };
 };
 
-export default async function ChainPage({ params }: Props) {
-  return redirect(`/${params.chainName}/worlds`);
+export default async function ChainPage({ params: { chainName } }: Props) {
+  return redirect(`/${chainName}/worlds`);
```

Comment:
thoughts on validating the chain name here and only redirect when valid?

Reply:
since chain names are known statically, I wonder if its helpful to use `getStaticPaths` on these routes?

Reply:
`getStaticPaths` only works with `pages` router while we are using `app` router which is the recommended way now and moving forward.

There is something similar in the `app` router called `generateStaticParams` which I think we can use the same way. However, I don't think we should make this page static given that the world addresses which we display in world entry page are dynamic. 

Now `getServerSideProps` inside `app` router is essentially replaced with `async` server-side component with a fetcher function inside. So based on that, the validation also is moved to the component itself. What I think would be best to do here is to actually move the validation inside `layout.tsx`. I already did that for `(explorer)/[chainName]/worlds/[worldAddress]/layout.tsx` but actually thinking that same could be done for `(explorer)/[chainName]/layout.tsx`(which doesn't exist yet but could be created for the purpose of validation). This way there's less repeating logic at least.